bookwyrm/templates/notifications/items/move.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'notifications/items/layout.html' %}
+
+{% load i18n %}
+{% load utilities %}
+
+{% block primary_link %}{% spaceless %}
+    {{ notification.related_object.local_path }}
+{% endspaceless %}{% endblock %}
+
+{% block icon %}
+    <span class="icon icon-local"></span>
+{% endblock %}
+
+{% block description %}
+    <!--
+        TODO: a user has a 'name' but not everything does, notably a book.
+        On the other hand, maybe we don't need to notify anyone if a book
```

Comment:
I don't think we would want to notify about books being merged, in part because it *ought* to be a routine administrative task and also it would potentially be a huge number of notifications